apps/summon-app/src/main.tsx

```diff
@@ -5,6 +5,8 @@ import * as ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 
 import App from './app/App';
+// FOR DEMONSTRATIOn
+// import { limitedNetworkTest } from './app/constants';
```

Comment:
Is this showing how this would be imported? Sorry if that's a really straight forward question ha